pylint/pyreverse/inspector.py

```diff
@@ -301,11 +319,14 @@ def _imported_module(self, node, mod_path, relative):
 
 
 def project_from_files(
-    files, func_wrapper=_astroid_wrapper, project_name="no name", black_list=("CVS",)
-):
+    files: list[str],
+    func_wrapper: _WrapperFuncT = _astroid_wrapper,
+    project_name: str = "no name",
+    black_list: tuple[str, ...] = ("CVS",),
```

Comment:
We might want to make this a constant and keep it up to date with `pylint`'s default value for the similar option?

Reply:
You mean the `black_list` / `ignore`? That would make sense imo.

Reply:
Yes!